deploy/charts/cert-manager/templates/servicemonitor.yaml

```diff
@@ -18,6 +18,12 @@ metadata:
     {{- with .Values.prometheus.servicemonitor.labels }}
     {{- toYaml . | nindent 4 }}
     {{- end }}
+{{- with .Values.prometheus.servicemonitor.annotations }}
+  {{- if . }}
+  annotations:
+    {{- toYaml . | nindent 4 }}
+  {{- end }}
+{{- end }}
```

Comment:
Can we write the annotations as the same labels strategy above to keep things consistent?

Reply:
Thanks for your review @JoshVanL. I've done something similar, while keeping the `if`. WDYT?

Reply:
Thanks!